columnphysics/icepack_tracers.F90

```diff
@@ -907,7 +907,7 @@ end subroutine icepack_query_tracer_indices
 
       subroutine icepack_write_tracer_indices(iounit)
 
-        integer, intent(in), optional :: iounit
```

Comment:
here `icepack_write_tracer_indices` is a public subroutine so we should re-run the autodocumentation script to update its interface in the doc to reflect the change of `iounit` from optional to mandatory.